### DISCUSSION TITLE
Is tag editor supposed to work?

### DISCUSSION THREAD

**Discussion by elXaptation** [rern/rAudio#161] [General]
First day using rAudio-1, its awesome.

Tried the tag editor. Seems to accept my changes in the editor box, but the tags actually do not change. Is this supposed to work? I'm kinda thinking it's a permissions problem. My source is a an NFS mount. It's unclear to me what UID/GID rAudio-1 is using when accessing files on the NFS. Anybody know? I'm trying to avoid a "maproot" or chmod777 solution.

**Comment by EarsofTin** [comment]
Tag editing works for me.  If I change the album Sgt.Pepper to Gen. Pepper on an individual track I get a new album in the mpd database and the metadata of the file is changed. 

Using CIFS mounted drive if that's relevant.
